packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -116,6 +109,14 @@ async function saveRecording(
     `Saving ${chalk.grey.bold(example)} with recording id ${chalk.yellow.bold(recordingId)}`
   );
 
+  if (!skipUpload) {
+    await uploadRecording(recordingId, {
```

Comment:
Why did you move this down again? I had moved it up to make it work for node examples.

Reply:
Because of the `console.log` message. I'll try moving them both up - is that Ok?